src/services/user/services/AdminUsers.js

```diff
@@ -12,8 +12,10 @@ const getCurrentUser = id => userModel.findById(id)
 	.lean()
 	.exec();
 
-const getAllUsers = (schoolId, roles, sortObject) => userModel.find({ schoolId, roles })
+const getAllUsers = (schoolId, roles, sortObject, skip = 0, limit = NaN) => userModel.find({ schoolId, roles })
```

Comment:
limit = 0 should be used to disable limit, below too - or not?

Reply:
Done 👍